rke/k8s_version_info.go

```diff
@@ -83,6 +83,14 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.3.7-rc0",
 			MinRKEVersion:     "1.0.7",
 		},
+		"v1.15.12-rancher1-1": {
+			MaxRancherVersion: "2.3.0",
+			MaxRKEVersion: "0.3.0",
```

Comment:
Shouldn't this be 0.2.99? Otherwise it will show up for 0.3.0?

Is this like min version where it will alway show up going forward?